src/App.tsx

```diff
@@ -2,108 +2,34 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+
+function getUser(userId: number): User | null {
+  const findUser = usersFromServer.find(user => user.id === userId);
+
+  return findUser || null;
+}
+
+function getComments(id: number): Comment[] {
+  return commentsFromServer
+    .filter(comment => comment.postId === id);
```

Comment:
[Checklist](https://github.com/mate-academy/react_static-list-of-posts/blob/master/README.md) the 5th point

Reply:
Rename these functions